pom.xml

```diff
@@ -798,7 +763,11 @@ under the License.
                             <importOrder>
                                 <order>org.apache.paimon,org.apache.paimon.shaded,,javax,java,scala,\#</order>
                             </importOrder>
-
+                            <replaceRegex>
+                                <name>Remove wildcard imports</name>
+                                <searchRegex>import\s+(static)*\s*[^\*\s]+\*;(\r\n|\r|\n)</searchRegex>
+                                <replacement>$1</replacement>
+                            </replaceRegex>
```

Comment:
This is not a very good solution, the part is to remove the import * if I understand correctly. If this is the case, after the contributor executes `mvn spotless:apply`, he submits the PR, and only when he executes the CI will he find that some classes are not imported.

Reply:
I'm afraid there is no good solution yet.
https://github.com/diffplug/spotless/issues/649.  Perhaps it is more appropriate now, thanks for help and i will close this pr . @zhuangchong